### PR TITLE
fix: revert async filter support at #15

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- `async filter` import at #15 will change all filter to async filter, which will cause a serious bug #17
- revert this feature

cc @beliefgp

### CODE REVIEW

**Comment by atian25** [comment]
since it's a bug, should we publish `patch` rather than `major break change` ?

**Comment by atian25** [comment]
since it's a bug, should we publish `patch` rather than `major break change` ?

cc @eggjs/core

File: lib/environment.js [hunk 1]
```diff
@@ -59,24 +56,10 @@ class NunjucksEnvironment extends nunjucks.Environment {
       const filterPath = path.join(unit.path, 'app/extend/filter.js');
       const filters = this.app.loader.loadFile(filterPath) || {};
       for (const key of Object.keys(filters)) {
-        this.addFilter(key, this[FILTER_WRAPPER](filters[key]), true);
-        // add filters to helper
-        if (this.ViewHelper.prototype[key] == null) {
```
Comment (fengmk2) [inline lib/environment.js]: 这个不是 https://github.com/eggjs/egg-view-nunjucks/pull/15/files 引入的吧

Comment (atian25) [inline lib/environment.js]: #16 为了修复 #15 而做的修改，一并回滚了。

**Comment by fengmk2** [comment]
This async filter feature is not working, wont break anything.

**Comment by atian25** [comment]
补上用例了

**Comment by atian25** [comment]
2.1.2

**Pull request merged**